docs/tutorial/maps.rst

```diff
@@ -4,63 +4,43 @@
 Maps
 ****
 
-Map objects in **sunpy** are two-dimensional data associated with a coordinate system.
-This class offers many advantages over using packages such as `astropy.io.fits` to open 2D solar data in Python.
-**sunpy** Maps be used with any image with two spatial axes that has FITS standard compliant metadata.
-There are also 19 sources (see :ref:`map` for a complete list) that provide instrument-specific information when plotting and in some cases interpret missing metadata when it is missing from the original file.
+In this section of the tutorial, you will learn about the `Map <sunpy.map.GenericMap>` object.
+Map objects hold two-dimensional data along with the accompanying metadata.
```

Comment:
```suggestion
Map objects hold two-dimensional data along with the accompanying coordinate information and metadata.
```

Reply:
i know the coordinate info is within the metadata, but it doesnt just carry this around it generates the wcs from it, so it may be good to explicitly say "coordinate info" or something along these lines here.

Reply:
My preference would be to keep the text at the front of the tutorial brief and to show the different components of the metadata in the subsequent sections (which I think this tutorial does). When I wrote "metadata" here, I'm lumping *all* metadata together, including information about the WCS. I'm not sure that distinction between non-WCS metadata and WCS metadata is important at this stage in the tutorial.